Common/Tools/TrackTuner.h

```diff
@@ -122,39 +134,51 @@ struct TrackTuner {
     /// +++ to be manually updated every time one adds a new parameter to the TrackTuner.h +++
     enum Pars : uint8_t { DebugInfo = 0,
                           UpdateTrackCovMat,
+                          UpdateTrackDCAs,
                           UpdateCurvature,
+                          UpdateCurvatureIU,
                           UpdatePulls,
-                          PathInputFile,
                           IsInputFileFromCCDB,
```

Comment:
in case the new `bool isInputFilePtSmearingFromCCDB` is introduced (as I said, unfortunately I do not see any other clean solution), then I'd change the name of this flag to `IsInputFileDcaSmearingFromCCDB` (here and everywhere else in the code).